manifests/qpid_router.pp

```diff
@@ -0,0 +1,89 @@
+# Constains certs specific configurations for qpid_dispatch
+class certs::qpid_router(
+  $hostname               = $::certs::node_fqdn,
+  $generate               = $::certs::generate,
+  $regenerate             = $::certs::regenerate,
+  $deploy                 = $::certs::deploy,
+  $server_cert            = $::certs::qpid_dispatch_server_cert,
+  $client_cert            = $::certs::qpid_dispatch_client_cert,
+  $server_key             = $::certs::qpid_dispatch_server_key,
+  $client_key             = $::certs::qpid_dispatch_client_key,
+  $owner                  = $::certs::qpid_dispatch_owner,
+  $group                  = $::certs::qpid_dispatch_group,
+) inherits certs::params {
+
+  $server_keypair = "${hostname}-qdrouterd-server"
```

Comment:
**Q**pid **D**ispatch **Router** **D**aemon, but I renamed it to qpid_router to be consistent with the class name.